root/static/scripts/release-editor/fields.js

```diff
@@ -447,6 +447,16 @@ class Medium {
                 self.release.earliestYear() < mediumFormatDate);
     });
     this.confirmedEarlyFormat = ko.observable(this.hasTooEarlyFormat());
+    this.hasStrangeDigitalPackaging = ko.computed(function () {
+      const isFormatDigital = self.formatID() &&
+                              self.formatID().toString() === '12';
+      return !!(isFormatDigital &&
+                self.release.packagingID() != null &&
+                self.release.packagingID().toString() !== '7'); // "None"
```

Comment:
Commenting all hard-coded values even though their meaning is obvious when writing and reviewing it at first:

```suggestion
      const isFormatDigital = self.formatID() &&
                              self.formatID().toString() === '12'; // "Digital Media"
      return !!(isFormatDigital &&
                self.release.packagingID() != null &&
                self.release.packagingID().toString() !== '7'); // "None"
```

It will help with finding them when they must be changed.